source/tutorials/tutorial-setup.txt

```diff
@@ -76,7 +76,7 @@ Set Up Your Development Environment with Docker
             .. code-block:: none
                :copyable: true
 
-               cd kafka-edu\docs-examples\mongodb-kafka-base\
+               cd {+win-sandbox-directory+}
```

Comment:
Q: I noticed there's no syntax highlighting on the Windows tab. Does it work if you change this to `code-block:: bash`, the way it is for Linux/Mac?

Reply:
I think this is not using "bash" as the language because it is not bash. If I remember correctly, the reason "none" is used is because the backslashes cause issues with syntax highlighting for bash. If you think syntax highlighting is necessary here, I can open a new ticket.